#request
nav_msgs/OccupancyGrid map_data
---
bool success